### ISSUE TITLE
For many co19 and language tests rastak does not notify the test framework of CompileTimeErrors

### ISSUE THREAD

**Issue by mkustermann** [dart-lang/rasta#78]
We expect rastak to report `CompileTimeError`s for programs if the specification demands it. Despite that, we also expect rastak to emit a Kernel IR (whether there were compile-time errors or not) and require the VM to report **also** a `CompileTimeError` for such tests.

Here is a list of tests which are failing with `MissingCompileTimeError`s:
```
co19/Language/Libraries_and_Scripts/top_level_syntax_t17
co19/Language/Libraries_and_Scripts/definition_syntax_t07
co19/Language/Libraries_and_Scripts/Imports/namespace_changes_t08
co19/Language/Libraries_and_Scripts/Imports/syntax_t22
co19/Language/Libraries_and_Scripts/Imports/syntax_t08
co19/Language/Libraries_and_Scripts/Imports/syntax_t39
co19/Language/Libraries_and_Scripts/Imports/syntax_t10
co19/Language/Libraries_and_Scripts/Imports/syntax_t20
co19/Language/Libraries_and_Scripts/Imports/syntax_t34
co19/Language/Libraries_and_Scripts/Imports/same_name_t17
co19/Language/Libraries_and_Scripts/Imports/syntax_t06
co19/Language/Libraries_and_Scripts/Imports/syntax_t07
co19/Language/Libraries_and_Scripts/Imports/syntax_t21
co19/Language/Libraries_and_Scripts/Imports/invalid_uri_t01
co19/Language/Libraries_and_Scripts/top_level_syntax_t02
co19/Language/Libraries_and_Scripts/Scripts/top_level_syntax_t05
co19/Language/Libraries_and_Scripts/Scripts/syntax_t01
co19/Language/Libraries_and_Scripts/Scripts/top_level_syntax_t17
co19/Language/Libraries_and_Scripts/Scripts/top_level_syntax_t13
co19/Language/Libraries_and_Scripts/Scripts/syntax_t13
co19/Language/Libraries_and_Scripts/Scripts/top_level_syntax_t02
co19/Language/Libraries_and_Scripts/definition_syntax_t20
co19/Language/Libraries_and_Scripts/Parts/compilation_t14
co19/Language/Libraries_and_Scripts/Parts/syntax_t03
co19/Language/Libraries_and_Scripts/Parts/compilation_t10
co19/Language/Libraries_and_Scripts/Parts/compilation_t13
co19/Language/Libraries_and_Scripts/Parts/compilation_t08
co19/Language/Libraries_and_Scripts/Parts/compilation_t02
co19/Language/Libraries_and_Scripts/Parts/compilation_t09
co19/Language/Libraries_and_Scripts/top_level_syntax_t12
co19/Language/Overview/Scoping/conflicting_names_t33
co19/Language/Overview/Scoping/conflicting_names_t12
co19/Language/Overview/Scoping/conflicting_names_t45
co19/Language/Overview/Scoping/conflicting_names_t19
co19/Language/Overview/Scoping/conflicting_names_t40
co19/Language/Overview/Scoping/conflicting_names_t43
co19/Language/Overview/Scoping/conflicting_names_t18
co19/Language/Overview/Scoping/conflicting_names_t37
co19/Language/Overview/Scoping/conflicting_names_t04
co19/Language/Overview/Scoping/conflicting_names_t35
co19/Language/Overview/Scoping/conflicting_names_t16
co19/Language/Overview/Scoping/conflicting_names_t11
co19/Language/Metadata/compilation_t06
co19/Language/Metadata/syntax_t08
co19/Language/Metadata/compilation_t11
co19/Language/Metadata/compilation_t08
co19/Language/Metadata/compilation_t07
co19/Language/Metadata/syntax_t07
co19/Language/Metadata/compilation_t09
co19/Language/Metadata/compilation_t04
co19/Language/Enums/restrictions_t04
co19/Language/Enums/restrictions_t07
co19/Language/Enums/syntax_t04
co19/Language/Enums/syntax_t05
co19/Language/Enums/restrictions_t02
co19/Language/Mixins/not_object_superclass_t01
co19/Language/Mixins/declaring_constructor_t01
co19/Language/Mixins/Mixin_Application/wrong_type_t02
co19/Language/Mixins/Mixin_Application/wrong_mixin_type_t01
co19/Language/Mixins/Mixin_Application/wrong_type_t01
co19/Language/Mixins/reference_to_super_t01
co19/Language/Mixins/deferred_t01
co19/Language/Generics/syntax_t13
co19/Language/Generics/syntax_t05
co19/Language/Generics/syntax_t15
co19/Language/Reference/Lexical_Rules/Comments/single_line_t02
co19/Language/Reference/Lexical_Rules/Comments/single_line_t04
co19/Language/Reference/Lexical_Rules/Comments/single_line_t01
co19/Language/Reference/Lexical_Rules/Comments/documentation_t11
co19/Language/Reference/Lexical_Rules/Comments/documentation_t04
co19/Language/Reference/Lexical_Rules/Comments/single_line_t03
co19/Language/Reference/Lexical_Rules/Comments/single_line_t08
co19/Language/Reference/Lexical_Rules/Comments/documentation_t10
co19/Language/Reference/Lexical_Rules/Comments/documentation_t06
co19/Language/Reference/Lexical_Rules/Comments/multi_line_t07
co19/Language/Reference/Lexical_Rules/whitespace_t07
co19/Language/Reference/Lexical_Rules/Reserved_Words/assert_t03
co19/Language/Reference/Lexical_Rules/Reserved_Words/while_t07
co19/Language/Reference/Lexical_Rules/Reserved_Words/break_t12
co19/Language/Reference/Lexical_Rules/Reserved_Words/whitespace_t04
co19/Language/Reference/Lexical_Rules/whitespace_t06
co19/Language/Variables/local_variable_t01
co19/Language/Variables/final_t07
co19/Language/Variables/syntax_t13
co19/Language/Variables/final_t03
co19/Language/Variables/final_t06
co19/Language/Variables/library_variable_t06
co19/Language/Variables/syntax_t21
co19/Language/Variables/syntax_t04
co19/Language/Types/Type_Declarations/syntax_t09
co19/Language/Types/Type_Declarations/self_reference_t08
co19/Language/Types/Type_Declarations/self_reference_t10
co19/Language/Types/Type_Declarations/self_reference_t02
co19/Language/Types/Type_Declarations/syntax_t07
co19/Language/Types/Type_Declarations/param_default_value_t02
co19/Language/Types/Type_Void/syntax_t01
co19/Language/Types/Type_Void/syntax_t10
co19/Language/Statements/Assert/syntax_t03
co19/Language/Statements/Assert/syntax_t01
co19/Language/Statements/Assert/syntax_t02
co19/Language/Statements/Assert/syntax_t04
co19/Language/Statements/While/syntax_t02
co19/Language/Statements/Break/label_t01
co19/Language/Statements/Break/label_t12
co19/Language/Statements/Break/label_t03
co19/Language/Statements/Switch/syntax_t08
co19/Language/Statements/Switch/equal_operator_t01
co19/Language/Statements/Switch/expressions_t04
co19/Language/Statements/Switch/expressions_t01
co19/Language/Statements/Switch/syntax_t07
co19/Language/Statements/Return/syntax_t03
co19/Language/Statements/Return/generative_constructor_t03
co19/Language/Statements/Local_Function_Declaration/syntax_t03
co19/Language/Statements/Local_Function_Declaration/reference_before_declaration_t03
co19/Language/Statements/Try/syntax_t16
co19/Language/Statements/Try/syntax_t18
co19/Language/Statements/Try/syntax_t17
co19/Language/Statements/Try/syntax_t09
co19/Language/Statements/Try/syntax_t19
co19/Language/Statements/Try/syntax_t10
co19/Language/Statements/Try/syntax_t13
co19/Language/Statements/Try/syntax_t02
co19/Language/Statements/Try/syntax_t06
co19/Language/Statements/Rethrow/on_catch_clause_t01
co19/Language/Statements/For/syntax_t09
co19/Language/Statements/Rethrow/on_catch_clause_t02
co19/Language/Statements/For/syntax_t02
co19/Language/Statements/For/syntax_t05
co19/Language/Statements/Local_Variable_Declaration/syntax_t11
co19/Language/Statements/Local_Variable_Declaration/syntax_t20
co19/Language/Statements/Local_Variable_Declaration/syntax_t04
co19/Language/Statements/Do/syntax_t03
co19/Language/Statements/Do/syntax_t02
co19/Language/Statements/Do/syntax_t07
co19/Language/Statements/If/syntax_t03
co19/Language/Statements/If/syntax_t01
co19/Language/Statements/If/syntax_t02
co19/Language/Statements/If/syntax_t06
co19/Language/Statements/If/syntax_t05
co19/Language/Statements/If/syntax_t04
co19/Language/Statements/Continue/syntax_t01
co19/Language/Statements/Continue/label_t16
co19/Language/Statements/Labels/syntax_t02
co19/Language/Statements/Labels/syntax_t07
co19/Language/Statements/Labels/syntax_t04
co19/Language/Statements/Expression_Statements/syntax_t12
co19/Language/Errors_and_Warnings/compile_error_t06
co19/Language/Errors_and_Warnings/compile_error_t03
co19/Language/Interfaces/Superinterfaces/superinterface_of_itself_t01
co19/Language/Interfaces/Superinterfaces/definition_t03
co19/Language/Classes/deсlarations_t33
co19/Language/Classes/deсlarations_t13
co19/Language/Classes/Instance_Variables/constant_t01
co19/Language/Classes/member_definition_t25
co19/Language/Classes/same_name_members_t01
co19/Language/Classes/deсlarations_t19
co19/Language/Classes/definition_t17
co19/Language/Classes/Instance_Methods/Operators/allowed_names_t08
co19/Language/Classes/Instance_Methods/Operators/arity_1_t18
co19/Language/Classes/Instance_Methods/Operators/arity_1_t13
co19/Language/Classes/Instance_Methods/Operators/allowed_names_t10
co19/Language/Classes/Instance_Methods/Operators/optional_parameter_t05
co19/Language/Classes/Instance_Methods/Operators/arity_2_t02
co19/Language/Classes/Instance_Methods/Operators/optional_parameter_t03
co19/Language/Classes/Instance_Methods/Operators/arity_1_t12
co19/Language/Classes/Instance_Methods/Operators/arity_1_t09
co19/Language/Classes/Instance_Methods/Operators/arity_1_t14
co19/Language/Classes/Instance_Methods/Operators/arity_1_t16
co19/Language/Classes/Instance_Methods/Operators/allowed_names_t21
co19/Language/Classes/Instance_Methods/Operators/arity_1_t06
co19/Language/Classes/Instance_Methods/Operators/arity_1_t10
co19/Language/Classes/Instance_Methods/Operators/arity_2_t01
co19/Language/Classes/Instance_Methods/Operators/arity_1_t04
co19/Language/Classes/Instance_Methods/Operators/arity_1_t07
co19/Language/Classes/Instance_Methods/Operators/arity_1_t02
co19/Language/Classes/Instance_Methods/Operators/arity_1_t17
co19/Language/Classes/Instance_Methods/Operators/arity_1_t01
co19/Language/Classes/Instance_Methods/Operators/optional_parameter_t08
co19/Language/Classes/Instance_Methods/Operators/syntax_t02
co19/Language/Classes/Instance_Methods/Operators/optional_parameter_t02
co19/Language/Classes/Instance_Methods/Operators/arity_1_t11
co19/Language/Classes/Instance_Methods/Operators/optional_parameter_t01
co19/Language/Classes/Instance_Methods/Operators/allowed_names_t12
co19/Language/Classes/Instance_Methods/Operators/optional_parameter_t06
co19/Language/Classes/Instance_Methods/Operators/arity_0_t02
co19/Language/Classes/Instance_Methods/Operators/arity_1_t15
co19/Language/Classes/Instance_Methods/Operators/optional_parameter_t04
co19/Language/Classes/Setters/parameter_t03
co19/Language/Classes/Setters/syntax_t03
co19/Language/Classes/Setters/parameter_t02
co19/Language/Classes/Setters/parameter_t07
co19/Language/Classes/Setters/static_setter_t04
co19/Language/Classes/definition_t18
co19/Language/Classes/method_definition_t08
co19/Language/Classes/Superclasses/wrong_superclass_t04
co19/Language/Classes/member_definition_t21
co19/Language/Classes/Getters/same_name_method_t01
co19/Language/Classes/Getters/same_name_method_t07
co19/Language/Classes/Getters/same_name_method_t12
co19/Language/Classes/Getters/same_name_method_t03
co19/Language/Classes/Getters/same_name_method_t08
co19/Language/Classes/Getters/syntax_t02
co19/Language/Classes/mixins_t02
co19/Language/Classes/Getters/syntax_t07
co19/Language/Classes/Getters/same_name_method_t04
co19/Language/Classes/deсlarations_t25
co19/Language/Classes/same_name_instance_and_static_members_t01
co19/Language/Classes/definition_t06
co19/Language/Classes/same_name_instance_and_static_members_t03
co19/Language/Classes/same_name_type_variable_t05
co19/Language/Classes/same_name_instance_and_static_members_t02
co19/Language/Classes/member_definition_t22
co19/Language/Classes/same_name_instance_and_static_members_t04
co19/Language/Classes/same_name_type_variable_t02
co19/Language/Classes/member_definition_t01
co19/Language/Classes/Constructors/Constant_Constructors/superinitializer_t01
co19/Language/Classes/Constructors/Constant_Constructors/potentially_constant_expression_t03
co19/Language/Classes/Constructors/Constant_Constructors/non_final_instance_variable_t01
co19/Language/Classes/Constructors/Constant_Constructors/syntax_t06
co19/Language/Classes/Constructors/Generative_Constructors/superinitializer_t01
co19/Language/Classes/Constructors/Generative_Constructors/superclass_constructor_t04
co19/Language/Classes/Constructors/Generative_Constructors/initializers_t12
co19/Language/Classes/Constructors/Generative_Constructors/initializers_t04
co19/Language/Classes/Constructors/Generative_Constructors/not_an_instance_variable_t03
co19/Language/Classes/Constructors/Generative_Constructors/formal_parameter_t06
co19/Language/Classes/Constructors/Generative_Constructors/redirection_t07
co19/Language/Classes/Constructors/Generative_Constructors/many_initializers_t03
co19/Language/Classes/Constructors/Generative_Constructors/superclass_constructor_t02
co19/Language/Classes/Constructors/Generative_Constructors/initializing_this_t02
co19/Language/Classes/Constructors/Generative_Constructors/superinitializer_t02
co19/Language/Classes/Constructors/Generative_Constructors/initializer_for_a_variable_t01
co19/Language/Classes/Constructors/Generative_Constructors/syntax_t05
co19/Language/Classes/Constructors/name_t01
co19/Language/Classes/Constructors/name_t02
co19/Language/Classes/Constructors/Factories/redirecting_to_itself_t04
co19/Language/Classes/Constructors/Factories/name_t04
co19/Language/Classes/Constructors/Factories/redirecting_constructor_t02
co19/Language/Classes/Constructors/Factories/name_t05
co19/Language/Classes/Constructors/Factories/const_modifier_t02
co19/Language/Classes/Constructors/Factories/name_t02
co19/Language/Classes/same_name_type_variable_t03
co19/Language/Classes/definition_t07
co19/Language/Classes/definition_t10
co19/Language/Classes/deсlarations_t14
co19/Language/Classes/same_name_members_t02
co19/Language/Classes/same_name_member_and_class_t04
co19/Language/Classes/same_name_type_variable_t08
co19/Language/Functions/Formal_Parameters/syntax_t17
co19/Language/Functions/Formal_Parameters/Required_Formals/syntax_t04
co19/Language/Functions/Formal_Parameters/declare_as_constant_t06
co19/Language/Functions/Formal_Parameters/declare_as_constant_t02
co19/Language/Functions/Formal_Parameters/syntax_t20
co19/Language/Functions/Formal_Parameters/declare_as_constant_t01
co19/Language/Functions/Formal_Parameters/syntax_t07
co19/Language/Functions/Formal_Parameters/Optional_Formals/syntax_t05
co19/Language/Functions/syntax_t32
co19/Language/Functions/syntax_t12
co19/Language/Expressions/Postfix_Expressions/syntax_t07
co19/Language/Expressions/Constants/ternary_operator_t02
co19/Language/Expressions/Constants/static_constant_t02
co19/Language/Expressions/Constants/no_other_constant_expressions_t09
co19/Language/Expressions/Constants/bitwise_operators_t03
co19/Language/Expressions/Constants/exception_t01
co19/Language/Expressions/Constants/literal_string_t05
co19/Language/Expressions/Constants/depending_on_itself_t03
co19/Language/Expressions/Constants/math_operators_t02
co19/Language/Expressions/Constants/constant_list_t02
co19/Language/Expressions/Constants/exception_t04
co19/Language/Expressions/Constants/literal_string_t03
co19/Language/Expressions/Constants/exception_t02
co19/Language/Expressions/Strings/multi_line_t13
co19/Language/Expressions/Strings/multi_line_t22
co19/Language/Expressions/Strings/multi_line_t03
co19/Language/Expressions/Strings/multi_line_t40
co19/Language/Expressions/Strings/multi_line_t42
co19/Language/Expressions/Strings/multi_line_t14
co19/Language/Expressions/Strings/string_literal_t31
co19/Language/Expressions/Strings/multi_line_t16
co19/Language/Expressions/Strings/multi_line_t05
co19/Language/Expressions/Strings/string_literal_t16
co19/Language/Expressions/Strings/string_literal_t03
co19/Language/Expressions/Strings/u_four_hex_digits_t04
co19/Language/Expressions/Strings/String_Interpolation/syntax_t03
co19/Language/Expressions/Strings/String_Interpolation/syntax_t08
co19/Language/Expressions/Strings/String_Interpolation/syntax_t11
co19/Language/Expressions/Strings/String_Interpolation/syntax_t14
co19/Language/Expressions/Strings/String_Interpolation/syntax_t15
co19/Language/Expressions/Strings/multi_line_t29
co19/Language/Expressions/Strings/multi_line_t36
co19/Language/Expressions/Strings/escape_hexadecimal_digits_t03
co19/Language/Expressions/Strings/string_literal_t29
co19/Language/Expressions/Strings/multi_line_t06
co19/Language/Expressions/Strings/string_literal_t10
co19/Language/Expressions/Strings/multi_line_t32
co19/Language/Expressions/Strings/escape_hexadecimal_digits_t02
co19/Language/Expressions/Function_Invocation/Unqualified_Invocation/invocation_t18
co19/Language/Expressions/Function_Invocation/Actual_Argument_List_Evaluation/syntax_t02
co19/Language/Expressions/Function_Invocation/Actual_Argument_List_Evaluation/syntax_t04
co19/Language/Expressions/Relational_Expressions/syntax_t16
co19/Language/Expressions/Relational_Expressions/syntax_t17
co19/Language/Expressions/Relational_Expressions/syntax_t09
co19/Language/Expressions/Relational_Expressions/syntax_t08
co19/Language/Expressions/Relational_Expressions/syntax_t10
co19/Language/Expressions/Relational_Expressions/syntax_t02
co19/Language/Expressions/Relational_Expressions/syntax_t14
co19/Language/Expressions/Relational_Expressions/syntax_t24
co19/Language/Expressions/Unary_Expressions/syntax_t09
co19/Language/Expressions/Unary_Expressions/syntax_t26
co19/Language/Expressions/Unary_Expressions/syntax_t06
co19/Language/Expressions/Unary_Expressions/syntax_t07
co19/Language/Expressions/Null/extend_or_implement_t01
co19/Language/Expressions/Assignable_Expressions/syntax_t16
co19/Language/Expressions/Assignable_Expressions/syntax_t17
co19/Language/Expressions/Assignable_Expressions/syntax_t22
co19/Language/Expressions/Assignable_Expressions/syntax_t25
co19/Language/Expressions/Assignable_Expressions/syntax_t20
co19/Language/Expressions/Assignable_Expressions/syntax_t10
co19/Language/Expressions/Assignable_Expressions/syntax_t24
co19/Language/Expressions/Conditional/syntax_t09
co19/Language/Expressions/Conditional/syntax_t06
co19/Language/Expressions/Conditional/syntax_t04
co19/Language/Expressions/Conditional/syntax_t05
co19/Language/Expressions/Multiplicative_Expressions/syntax_t26
co19/Language/Expressions/Multiplicative_Expressions/syntax_t03
co19/Language/Expressions/Multiplicative_Expressions/syntax_t25
co19/Language/Expressions/Multiplicative_Expressions/syntax_t19
co19/Language/Expressions/Multiplicative_Expressions/syntax_t02
co19/Language/Expressions/Multiplicative_Expressions/syntax_t06
co19/Language/Expressions/Multiplicative_Expressions/syntax_t05
co19/Language/Expressions/Lists/syntax_t03
co19/Language/Expressions/Lists/syntax_t02
co19/Language/Expressions/Lists/syntax_t07
co19/Language/Expressions/Lists/constant_list_t01
co19/Language/Expressions/Lists/syntax_t05
co19/Language/Expressions/If_null_Expressions/syntax_t06
co19/Language/Expressions/Logical_Boolean_Expressions/syntax_t09
co19/Language/Expressions/Logical_Boolean_Expressions/syntax_t08
co19/Language/Expressions/Logical_Boolean_Expressions/syntax_t07
co19/Language/Expressions/Logical_Boolean_Expressions/syntax_t06
co19/Language/Expressions/Throw/no_catch_clause_t04
co19/Language/Expressions/Throw/no_catch_clause_t01
co19/Language/Expressions/Throw/syntax_t02
co19/Language/Expressions/Throw/no_catch_clause_t06
co19/Language/Expressions/Identifier_Reference/syntax_t09
co19/Language/Expressions/Identifier_Reference/built_in_identifier_t29
co19/Language/Expressions/Identifier_Reference/built_in_identifier_t08
co19/Language/Expressions/Identifier_Reference/built_in_identifier_t23
co19/Language/Expressions/Identifier_Reference/built_in_identifier_t03
co19/Language/Expressions/Identifier_Reference/built_in_identifier_t19
co19/Language/Expressions/Identifier_Reference/syntax_t08
co19/Language/Expressions/Identifier_Reference/built_in_identifier_t31
co19/Language/Expressions/Identifier_Reference/built_in_not_dynamic_t01
co19/Language/Expressions/Identifier_Reference/built_in_identifier_t25
co19/Language/Expressions/Identifier_Reference/built_in_identifier_t04
co19/Language/Expressions/Identifier_Reference/built_in_identifier_t20
co19/Language/Expressions/Property_Extraction/General_Closurization/class_object_member_t06
co19/Language/Expressions/Property_Extraction/General_Closurization/class_object_member_t05
co19/Language/Expressions/Property_Extraction/General_Closurization/class_object_member_t01
co19/Language/Expressions/Property_Extraction/General_Closurization/class_object_member_t02
co19/Language/Expressions/Property_Extraction/Getter_Access_and_Method_Extraction/class_object_member_t06
co19/Language/Expressions/Type_Test/syntax_t03
co19/Language/Expressions/Type_Cast/syntax_t02
co19/Language/Expressions/Type_Cast/syntax_t06
co19/Language/Expressions/Type_Cast/syntax_t05
co19/Language/Expressions/Method_Invocation/3_Static_Invocation_A01_t04
co19/Language/Expressions/Method_Invocation/Cascaded_Invocations/syntax_t18
co19/Language/Expressions/Method_Invocation/Cascaded_Invocations/syntax_t17
co19/Language/Expressions/Method_Invocation/Cascaded_Invocations/syntax_t03
co19/Language/Expressions/Method_Invocation/Cascaded_Invocations/syntax_t09
co19/Language/Expressions/Method_Invocation/Cascaded_Invocations/syntax_t11
co19/Language/Expressions/Method_Invocation/Cascaded_Invocations/syntax_t07
co19/Language/Expressions/Method_Invocation/Cascaded_Invocations/syntax_t10
co19/Language/Expressions/Method_Invocation/Cascaded_Invocations/syntax_t04
co19/Language/Expressions/Method_Invocation/Ordinary_Invocation/syntax_t04
co19/Language/Expressions/Method_Invocation/Ordinary_Invocation/syntax_t05
co19/Language/Expressions/Method_Invocation/Super_Invocation/invocation_t02
co19/Language/Expressions/Method_Invocation/Super_Invocation/invocation_t03
co19/Language/Expressions/Method_Invocation/Super_Invocation/invocation_t05
co19/Language/Expressions/Method_Invocation/Super_Invocation/invocation_t06
co19/Language/Expressions/Method_Invocation/Super_Invocation/invocation_t01
co19/Language/Expressions/Method_Invocation/Super_Invocation/syntax_t05
co19/Language/Expressions/Method_Invocation/Super_Invocation/syntax_t04
co19/Language/Expressions/Instance_Creation/Const/exception_t01
co19/Language/Expressions/Instance_Creation/Const/constant_constructor_name_t03
co19/Language/Expressions/Instance_Creation/Const/accessibility_t03
co19/Language/Expressions/Instance_Creation/Const/constant_constructor_same_name_as_class_t02
co19/Language/Expressions/Instance_Creation/Const/syntax_t03
co19/Language/Expressions/Instance_Creation/Const/abstract_class_t03
co19/Language/Expressions/Instance_Creation/Const/abstract_class_t01
co19/Language/Expressions/Instance_Creation/Const/accessibility_t07
co19/Language/Expressions/Instance_Creation/malformed_or_malbounded_t04
co19/Language/Expressions/Instance_Creation/New/syntax_t03
co19/Language/Expressions/Instance_Creation/New/syntax_t02
co19/Language/Expressions/Instance_Creation/New/syntax_t05
co19/Language/Expressions/Shift/syntax_t08
co19/Language/Expressions/Shift/syntax_t06
co19/Language/Expressions/Shift/syntax_t02
co19/Language/Expressions/Shift/syntax_t07
co19/Language/Expressions/Shift/syntax_t04
co19/Language/Expressions/Maps/constant_map_t01
co19/Language/Expressions/Maps/key_value_equals_operator_t01
co19/Language/Expressions/Maps/constant_map_type_t01
co19/Language/Expressions/Maps/constant_map_key_value_t03
co19/Language/Expressions/Maps/syntax_t07
co19/Language/Expressions/Maps/syntax_t04
co19/Language/Expressions/Maps/syntax_t05
co19/Language/Expressions/This/placement_t08
co19/Language/Expressions/This/definition_t04
co19/Language/Expressions/This/placement_t01
co19/Language/Expressions/Equality/syntax_t02
co19/Language/Expressions/Booleans/extend_or_implement_t01
co19/Language/Expressions/Booleans/extend_or_implement_t02
co19/Language/Expressions/Numbers/syntax_t16
co19/Language/Expressions/Numbers/syntax_t17
co19/Language/Expressions/Numbers/extend_or_implement_double_t01
co19/Language/Expressions/Numbers/extend_or_implement_double_t02
co19/Language/Expressions/Numbers/syntax_t22
co19/Language/Expressions/Numbers/syntax_t29
co19/Language/Expressions/Bitwise_Expressions/syntax_t08
co19/Language/Expressions/Bitwise_Expressions/syntax_t10
co19/Language/Expressions/Bitwise_Expressions/syntax_t02
co19/Language/Expressions/Additive_Expressions/syntax_t16
co19/Language/Expressions/Additive_Expressions/syntax_t03
co19/Language/Expressions/Additive_Expressions/syntax_t02
co19/Language/Expressions/Additive_Expressions/syntax_t15
```

**Comment by peter-ahe-google** [comment]
Proposed fix: [CL 2204943003](https://codereview.chromium.org/2204943003).

**Comment by peter-ahe-google** [comment]
Repro:

```
==> bad_lib.dart <==
library bad_lib;

:

==> test1.dart <==
import 'bad_lib.dart';

main() {}

==> test2.dart <==
import 'bad_lib.dart';

main() {}
```

Then run:

```
echo $'test1.dart test1.dill\ntest2.dart test2.dill' | ./bin/rastak --bat
ch --
```
